src/gatsby/config.ts

```diff
@@ -79,6 +79,22 @@ const getPlugins = () => {
     "gatsby-plugin-sharp",
     "gatsby-plugin-sass",
     "gatsby-plugin-zeit-now",
+    {
+      resolve: "gatsby-plugin-google-gtag",
+      options: {
+        // You can add multiple tracking ids and a pageview event will be fired for all of them.
+        trackingIds: [
+          "UA-30327640-1", // Sentry
+          "UA-30327640-3", // www
```

Comment:
same question - is www desirable? was that how we had it before?

Reply:
This is how it was done before. Happy to remove if we don't need it.